core/auth/controller.go

```diff
@@ -0,0 +1,36 @@
+package auth
+
+import (
+	"context"
+
+	"flamingo.me/flamingo/v3/framework/web"
+)
+
+type WebDecorator struct {
```

Comment:
Should this be something like middleware? If yes, I would maybe rename it so, just for purpose of consistency (we already use that naming, and it's a kind of standard in routing go frameworks).

Reply:
Yes, middleware, didn't remember the wording -.-*